src/artemis/devices/CTAB.py

```diff
@@ -0,0 +1,24 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsMotor, EpicsSignalRO
+
+
+class CTAB(Device):
+    """Basic collimantion table (CTAB) device for motion plus the motion disable signal when laser curtain triggered and hutch not locked"""
+
+    inboard_y: EpicsMotor = Cpt(EpicsMotor, "-MO-TABLE-01:INBOARDY")
```

Comment:
Nit: Do you know if there's anywhere this co-ord system is documented? I can write some down if not